GUIDE.md

```diff
@@ -0,0 +1,181 @@
+example: it's great to include *brief* example code in the reference
+documentation, but it is not so great to include a full program there.
+
+To summarize:
+
+* Structure is of paramount importance.
+* Decide on an **audience** and an **idea** you want to convey to that
+audience.
+* If the reader is to follow a process to get to a goal, it's a **guide.**
+* If what you're writing conveys a concept to help the reader in decision
+making, it's a **topic.**
+* When the reader is consulting your document to find a specific piece of
+information, it's **reference.**
+* Each document should clearly convey *one* idea to *one* audience. All
+strategies are in service of that goal.
+
+## How We Write
+
+* Documents are written in markdown files. 
+* Those files should be written in **`lowercase-with-dashes.md`.**
+  * Underscores in filenames are allowed only when they are present in the
+    topic the document will describe (e.g., `child_process`.)
+  * Filenames should be **lowercase**.
+  * Older files may use the `.markdown` extension. These may be ported to `.md`
+    at will. **Prefer `.md` for all new documents.**
+* Documents should be word-wrapped at 80 characters.
+* Documents should not contain trailing whitespace.
+* Mechanical issues, like spelling and grammar, should be identified by tools,
+  insofar as is possible. If not caught by a tool, they should be pointed out by
+  human reviewers.
+* US spelling is preferred. "Capitalize" vs. "Capitalise", "color" vs.
+  "colour", etc.
+* Though controversial, the [Oxford comma][] is preferred for clarity's sake.
+* Generally avoid personal pronouns in reference documentation ("I", "you",
+  "we".)
+  * Pronouns are acceptable in more colloquial documentation, like guides.
+  * Use **gender-neutral pronouns** and **mass nouns**. Non-comprehensive
+    examples:
+    * **OK**: "they", "their", "them", "folks", "people", "developers"
+    * **NOT OK**: "his", "hers", "him", "her", "guys", "dudes".
+* Place end-of-sentence punctuation inside wrapping elements — periods go
+  inside parentheses and quotes, not after.
+* Documents must start with a level-one heading. An example document will be
+  linked here eventually.
+* Prefer affixing links to inlining links — prefer `[a link][]` to
+  `[a link](google dot com)`.
+* When documenting APIs, note the version the API was introduced in at
+  the end of the section. If an API has been deprecated, also note the first
+  version that the API appeared deprecated in.
+* When using dashes, use emdashes ("—", Ctrl+Alt+"-" on OSX) surrounded by
+  spaces, per the New York Times usage.
+* Including assets:
+  * If you wish to add an illustration or full program, add it to the
+    appropriate sub-directory in the `assets/` dir.
+  * Link to it like so: `[Asset](/assets/{subdir}/{filename})` for file-based
+    assets, and `![Asset](/assets/{subdir}/{filename})` for image-based assets.
+* For code blocks:
+  * Use language aware fences.
```

Comment:
So, fun story: I found a GitHub markdown bug this way!

I attempted to add a "```markdown" block to illustrate how to use code blocks, but it broke all of the links! I'll try again, but if it breaks the links again I'm going to give up on that one :)